backend/yugioh/management/commands/seed.py

```diff
@@ -0,0 +1,244 @@
+
+                    try:
+                        rarity_object, is_rarity_created = YugiohCardRarity.objects.get_or_create(rarity=rarity,
+                                                                                                  rarity_code=rarity_code)
+                    except IntegrityError:
+                        code_generation = rarity.split()
+                        for word in code_generation:
+                            rarity_code += word[1]
+                        rarity_object, is_rarity_created = YugiohCardRarity.objects.get_or_create(rarity=rarity,
+                                                                                                  rarity_code=rarity_code)
+
+                    card_set_object, is_cardset_created = YugiohCardSet.objects.get_or_create(
+                        card_set_name=card_set_name,
+                        set_code=card_set_code)
+
+                    if is_rarity_created:
+                        self.stdout.write(f'Rarity {rarity} does not exist... Creating')
+                        logging.info(f'Rarity * {rarity} * does not exist... Creating')
+                    else:
+                        self.stdout.write(f'{rarity} exist... Skipping')
+                        logging.info(f'{rarity} exist... Skipping')
+
+                    if is_cardset_created:
+                        self.stdout.write(f'Cardset {card_set_name} does not exist... Creating')
+                        logging.info(f'Cardset {card_set_name} does not exist... Creating')
+                    else:
+                        self.stdout.write(f'Cardset {card_set_name} exist... Skipping')
+                        logging.info(f'Cardset {card_set_name} exist... Skipping')
+
+                    existing_card = YugiohCardInSet.objects.filter(
+                        rarity=rarity_object,
+                        set=card_set_object,
+                        yugioh_card=yugioh_card_object
+                    ).first()
+
+                    if not existing_card:
+                        YugiohCardInSet.objects.create(rarity=rarity_object, set=card_set_object,
+                                                       yugioh_card=yugioh_card_object)
+
+                        imported_set += 1
+
+                    else:
+                        self.stdout.write(self.style.ERROR(f'Card {yugioh_card_object} '
+                                                           f'{card_set_object} {rarity_object}... Skipping'))
+                        logging.info(f'Card {yugioh_card_object} with '
+                                     f'{card_set_object} {rarity_object}... Skipping')
+            except KeyError:
+                logging.info(f'No sets for {card_name}')
+
+        logging.info("--------------")
+        logging.info(f"Processed cards -> {processed_cards}")
+        logging.info(f"    - Imported cards -> {imported_card}")
+        logging.info(f"Processed sets -> {processed_sets}")
```

Comment:
The `processed_sets` value actually represents the `CardInSet` objects. The processed sets are ~980.

Reply:
it is fixed now